addons/binding/org.openhab.binding.rotelra1x/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
```

Comment:
Seems like this should be changed somehow, since it references Java 1.7,  but I'm not sure what to replace it with.

Reply:
Just replace by 1.8

Reply:
Ok, thanks!